.github/workflows/goreleaser.yml

```
@@ -14,7 +14,7 @@ jobs:
       - name: setup-go
         uses: actions/setup-go@v5
         with:
-          go-version: "1.20"
+          go-version: "1.22"
```

This will use the `go.mod` version, and save updating it all the time.
```suggestion
          go-version-file: 'go.mod'
```

---

Good idea!